flink-end-to-end-tests/test-scripts/test_pyflink.sh

```diff
@@ -223,13 +223,23 @@ function read_msg_from_kafka {
 }
 
 function cat_jm_logs {
-     local log_file_name=${3:-standalonesession}
-     cat $FLINK_LOG_DIR/*$log_file_name*.log
+  cat_logs_in_order "standalonesession"
 }
 
 function cat_tm_logs {
-	local logfile="$FLINK_LOG_DIR/flink*taskexecutor*log"
```

Comment:
@zentol When preparing the backports I noticed that I missed a few occurrences because I grepped for ".log" instead of "log" when looking for code snippets to change in any `*sh` file. Like this one above